src/test/java/org/apache/sysds/test/functions/builtin/BuiltinMulticlassSVMTest.java

```diff
@@ -53,38 +52,38 @@ public void setUp() {
 
 	@Test
 	public void testMSVMDense() {
-		runMSVMTest(false, false, num_classes, eps, 1.0, max_iter, LopProperties.ExecType.CP);
+		runMSVMTest(false, false, eps, 1.0, max_iter, LopProperties.ExecType.CP);
 	}
 	@Test
 	public void testMSVMSparse() {
-		runMSVMTest(true, false, num_classes, eps, 1.0, max_iter, LopProperties.ExecType.CP);
+		runMSVMTest(true, false, eps, 1.0, max_iter, LopProperties.ExecType.CP);
 	}
 	@Test
 	public void testMSVMInterceptSpark() {
-		runMSVMTest(true,true, num_classes, eps, 1.0, max_iter, LopProperties.ExecType.SPARK);
+		runMSVMTest(true,true, eps, 1.0, max_iter, LopProperties.ExecType.SPARK);
 	}
 
 	@Test
 	public void testMSVMSparseLambda2() {
-		runMSVMTest(true,true, num_classes, eps,2.0, max_iter, LopProperties.ExecType.CP);
+		runMSVMTest(true,true, eps,2.0, max_iter, LopProperties.ExecType.CP);
 	}
 	@Test
 	public void testMSVMSparseLambda100CP() {
-		runMSVMTest(true,true, num_classes, 1, 100, max_iter, LopProperties.ExecType.CP);
+		runMSVMTest(true,true, 1, 100, max_iter, LopProperties.ExecType.CP);
 	}
 	@Test
 	public void testMSVMSparseLambda100Spark() {
-		runMSVMTest(true,true, num_classes, 1, 100, max_iter, LopProperties.ExecType.SPARK);
+		runMSVMTest(true,true, 1, 100, max_iter, LopProperties.ExecType.SPARK);
 	}
 	@Test
 	public void testMSVMIteration() {
-		runMSVMTest(true,true, num_classes, 1, 2.0, 100, LopProperties.ExecType.CP);
+		runMSVMTest(true,true, 1, 2.0, 100, LopProperties.ExecType.CP);
 	}
 	@Test
 	public void testMSVMDenseIntercept() {
-		runMSVMTest(false,true, num_classes, eps, 1.0, max_iter, LopProperties.ExecType.CP);
+		runMSVMTest(false,true, eps, 1.0, max_iter, LopProperties.ExecType.CP);
 	}
-	private void runMSVMTest(boolean sparse, boolean  intercept, int classes, double eps,
+	private void runMSVMTest(boolean sparse, boolean  intercept, double eps,
 							  double lambda, int run, LopProperties.ExecType instType)
```

Comment:
again, personally I would use a single indentation level for parameters (as people read from left to right) but I don't have a strong opinion here.

Reply:
formatted file with our for matter, I tried to avoid it because it is not "my" file. But i need to add some testing for the new type of input anyway, so further work incoming.

Testing after code :scream:.